hybrid/compartment_examples/call_sentry_and_unbound_pcc/main.c

```diff
@@ -0,0 +1,55 @@
+/***
+ * This example showcases how a restricted PCC, after being unbound to
```

Comment:
Silly question: what does "unbound" mean?

Reply:
I mean it here as changing the "bounds" fields within the capability to (presumably, have not checked) cover the whole memory space, rather than the limited space we constrain it to. Which I believe happens as a side-effect of unsealing, overwriting our set `PCC` via a jump instruction.

Reply:
Let me try rephrasing what you've said and see if it makes sense.

If I call a sealed function `f` via a capability `C`, the `PCC` will be set to `C`'s bounds during execution. After `f` returns, the `PCC` remains set to whatever `C`'s bounds were.

Is that correct?

Reply:
I have confirmed this and you are correct; the `PCC` is set to the bounds set within the sentry. Which now puts the issue that when we return from `C` code back to assembly, we can't execute anymore unless the PCC contained within the sentry covers that code...